src/Flex.js

```diff
@@ -4,10 +4,7 @@ import { space, width, color } from 'styled-system'
 
 const Flex = styled.div([], {
   display: 'flex',
-  justifyContent: 'center',
-  '@media print': {
-    display: 'none'
-  }
```

Comment:
Sorry for the delay! This isn't ideal, but this line is what hides the navigation dots during PDF export, so it would be good to handle that somehow with this change